### ISSUE TITLE
Add kubernetes deployments

### ISSUE THREAD

**Issue by meysamhadeli** [meysamhadeli/booking-microservices-nestjs#9]
- Create separate deployment file for each microservices
- Create separate service file for each microservices
- Use Ingress as a revers proxy

**Comment by shantanuborade007** [comment]
hey Sam !
can you assign me this enhancement , i can work on this

**Comment by meysamhadeli** [comment]
> hey Sam ! can you assign me this enhancement , i can work on this

@shantanuborade007 Thanks, You can also take a look a implementation of Kubernetesin my other project and I want to have the same structure here: https://github.com/meysamhadeli/booking-microservices/blob/main/deployments/kubernetes/booking-microservices.yml

> [!NOTE]
> If it is possible to implement it with a `Helm Chart`, that would be great :)

**Comment by shantanuborade007** [comment]
yep sure , I do have some experience working with helm , I'll implement accoriding to existing yml format.
